pygac/reader.py

```diff
@@ -707,7 +711,26 @@ def get_tle_lines(self):
         self.tle_lines = tle1, tle2
         return tle1, tle2
 
-    def get_sat_angles_without_tle(self):
+    def get_sat_angles(self):
+        """Get satellite angles.
+
+        Returns:
+            Azimuth, elevation (degrees)
+        """
+        try:
+            return self._get_sat_angles_with_tle()
+        except NoTLEData:
```

Comment:
Should we log a warning when this happens?

Reply:
Good idea!